ntbs-service/Pages/Notifications/Edit/_PaginationPartial.cshtml

```diff
@@ -1,7 +1,7 @@
 <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
     <ul class="nhsuk-list nhsuk-pagination__list">
 
-        @if (!(ViewData["FirstPage"] as bool?) ?? true) 
+        @if (ViewData["PreviousPage"] != null )

```

Comment:
I suppose the "First Page" item should be removed from the Patient Details page?